language-plutus-core/plutus-ir-test/types/bindDataTypeNormalize.golden

```diff
@@ -0,0 +1 @@
+[Maybe_0 (con integer)]
```

Comment:
Ah this example was not about escaping types, it was mostly to force the plc-typechecker where the pir typechecker would succeed. It is because the pir-compiler does not do type normalization inside the dataconstructors.

Reply:
changed it to finish the pattern match. revealed a typing bug which I fixed.